bin/generate_crossref_doi_metadata.py

```diff
@@ -188,7 +188,10 @@ def main(volumes):
                         make_simple_element("surname", text=name_part.text, parent=pn)
 
         if editor_index == 0:
-            print("FATAL: Found no editors", file=sys.stderr)
+            print(
+                f"WARNING: Found no editors for volume {full_volume_id}, skipping",
+                file=sys.stderr,
+            )
```

Comment:
I'd recommend `logging.warning()` or `logging.error()` over `print` in the future.